docs/changelog.md

```diff
@@ -12,6 +12,8 @@ permalink: /changelog/
 * [Gecko](https://github.com/mozilla-mobile/android-components/blob/master/buildSrc/src/main/java/Gecko.kt)
 * [Configuration](https://github.com/mozilla-mobile/android-components/blob/master/.config.yml)
 
+* **browser-state**
+  * ⚠️ **This is a breaking change**: Renames `DownloadState` doesn't support [Parcelable](https://developer.android.com/reference/android/os/Parcelable) anymore.
```

Comment:
"Renames"?

Reply:
A residue of a copy paste 🤦 . Fixing it :)